terminus-ui/autocomplete/src/autocomplete.component.ts

```diff
@@ -45,7 +43,7 @@ import { TsStyleThemeTypes } from '@terminus/ui/utilities';
 import {
   BehaviorSubject,
   of,
-  Subject,
+  Subject, throwError,
```

Comment:
Might want two lines...

Reply:
We really need to write a lint rule for this.. 😞

Reply:
let me fix. I think this imort isn't needed either..

Reply:
fixed!